Create 2021-09-02-RenesasofficialsupportedHW.md (#348)
micro-ROS/micro-ROS.github.io: _posts/2021-09-02-RenesasofficialsupportedHW.md

---

> title: Renesas and eProsima Simplify Development Of Professional Robotics Applications On RA MCUs With micro-ROS Development Framework

Reviewer: ```suggestion
title: Renesas and eProsima Simplify Development of Professional Robotics Applications on RA MCUs with micro-ROS Development Framework
```

author: Maria Merlan
---

> Agreement Facilitates Adoption of Robotics in Industrial and IoT Sectors; EK-RA6M5 Evaluation Kit Now Official Supported micro-ROS Development Board.

Reviewer: ```suggestion
Agreement facilitates adoption of robotics in Industrial and IoT sectors; EK-RA6M5 Evaluation Kit is now an officially supported micro-ROS development board.
```

> Renesas is teaming with eProsima to facilitate the adoption of Robotics in the Industrial and IoT sectors. [Renesas RA6M5 based micro-ROS project](https://github.com/micro-ROS/micro_ros_renesas2estudio_component) is a disruptive solution that provides an eclipse environment that covers the full development cycle of embedded applications in ROS 2. This will strongly impact on reducing the time-to-market of robotics solutions across many different sectors. 

Reviewer: ```suggestion
Renesas is teaming with eProsima to facilitate the adoption of robotics in the industrial and IoT sectors. [Renesas RA6M5 based micro-ROS project](https://github.com/micro-ROS/micro_ros_renesas2estudio_component) is a disruptive solution that provides an Eclipse environment that covers the full development cycle of embedded applications in ROS 2. This will strongly impact on reducing the time-to-market of robotics solutions across many different sectors. 
```

[This achievement](https://www.renesas.com/eu/en/about/press-room/renesas-and-eprosima-simplify-development-professional-robotics-applications-ra-mcus-micro-ros) comes from joint collaboration between Renesas Electronics Corporation (TSE:6723), a premier supplier of advanced semiconductor solutions and eProsima, the middleware expert SME. 

The [Renesas RA6M5](https://www.renesas.com/us/en/products/microcontrollers-microprocessors/ra-cortex-m-mcus/ek-ra6m5-evaluation-kit-ra6m5-mcu-group) based micro-ROS project offers HIL (hardware in the loop) testing CD/CI and integration with the e² studio, an Eclipse-based integrated development environment (IDE) that enables developers a professional embedded software environment.

> The objective of this HW & SW combo project is to provide a comprehensive tool that covers the full cycle of SW development in embedded systems that target a wide range of applications: Service robots for Logistic & Warehouse, Defense & Security, Agriculture and Healthcare. 

Reviewer: ```suggestion
The objective of this HW & SW combo project is to provide a comprehensive tool that covers the full cycle of SW development in embedded systems that target a wide range of applications: service robots for logistic & warehouse, defense & security, agriculture and healthcare. 
```

eProsima: “Achieving cost-effective development depends on a common platform around which hardware and software providers emerge,” said Jaime Martin Losa, eProsima CEO. “micro-ROS represents the tool that bridges the gap between microcontrollers and ROS 2, expanding the scope of new robotic applications based on embedded devices.”

Renesas: “Robotics is becoming increasingly important as Industry 4.0 and Industrial IoT gain steam,” said Roger Wendelken, Senior Vice President in the IoT and Infrastructure Business Unit at Renesas. “The performance and feature set of the RA Family make it the best choice for robotic applications, and we’re pleased to work with eProsima to deliver solutions to our diverse customer base.”

> Learn how to integrate micro-ROS with RA Family in your high-level system with [Getting Started Guide and Demo videos](https://www.youtube.com/playlist?list=PL-Kh3H15FsNTD8A8wD0_iMnvhDsqw-rga)

Reviewer: ```suggestion
Learn how to integrate micro-ROS with RA Family in your high-level system with [Getting Started Guide and Demo videos](https://www.youtube.com/playlist?list=PL-Kh3H15FsNTD8A8wD0_iMnvhDsqw-rga).
```

> - GITHUB SOURCES:

Reviewer: ```suggestion

Source code at GitHub:
```
